connectors/connector.go

```diff
@@ -1,6 +1,16 @@
 package connectors
 
+import (
+	"github.com/nicksdlc/macaw/model"
+)
+
+// Connector is an interface the represents connector to external source
 type Connector interface {
 	Close() error
+
 	Post(body string) error
```

Comment:
What if request also contains some metadata? Like headers in http or RMQ? May be wrap in some struct like "Message"?